apps/lend/src/hooks/usePageOnMount.ts

```diff
@@ -32,26 +34,44 @@ function usePageOnMount(params: Params, location: Location, navigate: NavigateFu
   const walletSignerAddress = getWalletSignerAddress(wallet)
   const parsedParams = parseParams(params, chainIdNotRequired)
 
+  const handleReconnectWallet = useCallback(
```

Comment:
You can move this and similar functions to the `onboard-helper` package, wrap them in a hook and use it.
```ts
const useOnboardWallet = () => {
  ...
  const handleReconnect = useCallback( ... )
  ...
  return [wallet, handleReconnect, ... ]
}
```